scripts/qa_crowbarsetup.sh

```diff
@@ -3637,7 +3637,7 @@ function oncontroller_manila_generic_driver_setup()
         manila_tenant_vm_ip=`oncontroller_manila_service_instance_get_floating_ip`
     else
         fixed_net_id=`neutron net-show fixed -f value -c id`
-        timeout 10m nova boot --poll --flavor 100 --image manila-service-image \
+        timeout 10m nova boot --insecure --poll --flavor 100 --image manila-service-image \
```

Comment:
It would be even better if you updated the command to the `openstack` CLI :smile: